pages/concepts/ai-engine/ai-engine-personalities.mdx

```diff
@@ -0,0 +1,58 @@
+    2. **Understanding the request**: the AI Engine employs LLMs to analyze the user's request, grasping the intent and identifying the necessary actions.
+    3. **Task delegation**: based on the analysis, the AI Engine breaks down the user's request into executable tasks.
+    4. **Agent coordination**: the AI Engine then orchestrates a network of agents registered on the Fetch Network to fulfill those tasks, ultimately achieving the user's objective.
+
+AI Engine personalities are not primarily about user experience or preference. Instead, they represent different versions or iterations of the AI Engine itself, each with its own set of features and capabilities.
+
+The primary reason for introducing different personalities is to manage the development and rollout of new features without disrupting existing functionality. By creating a new personality, developers can experiment with and test new features without affecting the core system. Once these features are stable and mature, they can be integrated into the default personality.
+
+## A Journey through personality types evolution
+
+The following AI Engine personality types are available for users: **Talkative V3** and **NextGeneration**.
+
+    - With version **v0.22**, the NextGeneration personality received significant updates, including support for classical UAgentResponse type agents, secondary function execution, and interactive UI elements. Notably, Talkative V3 was set to be replaced by NextGeneration in future updates.
+    - Version **v0.23** introduced enhanced session-level memory, tracking credit usage, parallel execution in sync worker, and session termination control within the NextGeneration personality.
+    - Version **v0.24** deployed **NextGeneration personality as the default**, and thus replacing **Talkative V3** personality type. This version focuses on significant stability and scalability improvements, supporting millions of agents. This version also introduced immediate agent registration, enhanced message reliability, UI enhancements, privacy compliance, robust timeout and monitoring systems, infrastructure upgrades, and database and deployment improvements.
+
+## Exploring the Feature Set of AI Engine Personalities
+
+Here's a breakdown of the key functionalities offered by each personality:
+
+### Talkative V3 - AI Engine V1
+
+    **Focus**: General conversation and task handling.
+
+    **Key Functionalities**:
+        - Enhanced task recommendations for improved efficiency.
+        - Improved context building for natural conversation flow.
+        - Agent health checks to ensure optimal performance.
+
+### NextGeneration - AI Engine V2
+
+    **Focus**: Broad conversational capabilities, advanced functionalities, and scalability.
+
+    **Key Functionalities**:
+        - Interacts with websites and PDFs directly.
```

Comment:
it could be an example for the to be created first bullet point (see my comment below) and the "directly" might be deleted since AI Engine doesn't directly interact with websites and PDFs but through agents with dialogue protocol - so it is the dialogue protocol that enables multiple rounds of messaging an agent (that uses the Protocol class) and getting a response from the agent (for example multiple question-answering related to a website or PDF)